code/src/main/kotlin/com/expediagroup/sdk/core/logging/common/LoggableContentTypes.kt

```diff
@@ -8,13 +8,17 @@ import com.expediagroup.sdk.core.http.MediaType
  * can be logged based on their MIME types.
  */
 val LOGGABLE_CONTENT_TYPES = listOf(
-    MediaType.APPLICATION_JSON,
-    MediaType.TEXT_PLAIN,
-    MediaType.TEXT_HTML,
-    MediaType.APPLICATION_XML,
-    MediaType.parse("text/*"), // Wildcard to include any text-based types
-    MediaType.parse("application/*+json"), // Matches custom JSON-based media types like application/.api+json
-    MediaType.APPLICATION_FORM_URLENCODED
+    MediaType.of("text", "plain"),
```

Comment:
Should we define `text` and `application` as `const`ants?

Reply:
I didn't want to introduce a new constant for it to avoid possible confusion when you want to import the actual TEXT_PLAIN from MediaType.

Reply:
We can have `private const val TEXT = "text"` and the same for others. It would be a good idea for `CommonMediaTypes` as well.